testutil/network/network.go

```diff
@@ -117,6 +115,7 @@ func DefaultApplicationModuleGenesisState(t *testing.T, n int) *apptypes.Genesis
 				},
 			},
 		}
+		// NB: Might be useful for development but commented to cause bugs if empty fields cause business logic issues
```

Comment:
I am unsure on what this means? Maybe this is a good time for us to enforce the usage of the `(gogoproto.nullable) = false` proto option?

Reply:
Replacing with `// TODO_CONSIDERATION: Evaluate whether we need `nullify.Fill` or if we should enforce `(gogoproto.nullable) = false` everywhere` everywhere